source/ptreferential/ptreferential.cpp

```diff
@@ -106,7 +107,7 @@ struct select_r: qi::grammar<Iterator, std::vector<Filter>(), qi::space_type>
             [qi::_val = boost::phoenix::construct<Filter>(qi::_1, qi::_2)];
         after_clause = ("AFTER(" >> text >> ')')[qi::_val = boost::phoenix::construct<Filter>(qi::_1)];
         args_clause = (word|escaped_string|bracket_string)[boost::phoenix::push_back(qi::_val, qi::_1)] % ',';
-        method_clause = (word >> "." >> word >> "(" >> args_clause >> ")")
+        method_clause = (word >> "." >> word >> "("  >> -(args_clause) >> ")" )
```

Comment:
same here, I think it would be nice to add a comment explaining that args_clause is optional